mcp_zammad/models.py

```
@@ -3,7 +3,36 @@
+        article_id: int,
+        attachment_id: int,
+        original_error: Exception,
+    ) -> None:
+        """Initialize the exception with context."""
+        self.ticket_id = ticket_id
+        self.article_id = article_id
+        self.attachment_id = attachment_id
+        self.original_error = original_error
+        self.message = (
+            f"Failed to download attachment {attachment_id} for ticket {ticket_id} "
+            f"article {article_id}: {str(original_error)}"
```

Ruff (0.14.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)